.travis.yml

```diff
@@ -24,6 +24,7 @@ deploy:
     - release/goss-linux-386
     - release/goss-linux-arm
     - extras/dgoss/dgoss
+    - extras/kgoss/kgoss
```

Comment:
Can you drop it from the releases. Only dgoss is released for now.

Everything else can be referenced by the README.

In the future I'll look for a consistent way to handle dgoss, dcgoss, kgoss, podman Goss tool, etc..

Reply:
If you don't get around to it. I'll merge this and follow it up with a one-liner change. I'm just out of town and won't get to it until later in the week.

Reply:
No problem @aelsabbahy, I just removed that line. Makes sense not to include in a release at this point, looking forward to getting a bit more feedback.